src/Qir/Execution/QirCommandLineTool/Program.cs

```diff
@@ -0,0 +1,156 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Help;
+using System.CommandLine.Invocation;
+using System.CommandLine.Parsing;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Quantum.Qir.Tools;
+
+namespace CommandLineCompiler
```

Comment:
Should this namespace be prefixed with `Microsoft.Quantum.`?

Reply:
done